lib/percy/capybara.rb

```diff
@@ -54,8 +54,10 @@ def self.disable!
     end
 
     def self.use_loader(loader, options = {})
-      capybara_client.loader = loader
-      capybara_client.loader_options = options
+      client = capybara_client(loader: loader)
```

Comment:
_Maybe_. Will have to carefully look over what folks currently pass into `use_loader`.